examples/python/trigger-pagerduty-incident/handler/handler.py

```diff
@@ -63,94 +69,112 @@ def invoke(self,obj):
         Returns:
             FaaSResponse -- status code and message
         """
-        obj['routing_key'] = self.routingkey
-        obj['event_action'] = self.event_action
         try:
             resp = self.session.post(PAGERDUTY_API_PATH,json=obj)
             resp.raise_for_status()
-            print(resp.text)
+            debug(f'{bgc.OKGREEN}Response Body > {bgc.ENDC}{resp.text}')
             resp_body = json.loads(resp.text)
-            return FaaSResponse('200', 'Successfully {0}ed event action with dedup_key: {1}'.format(self.event_action, resp_body['dedup_key']))
+            return FaaSResponse('200', 'Successfully triggered event action with dedup_key: {0}'.format(resp_body['dedup_key']))
```

Comment:
Can we align this with the error message, e.g. 'Successfully invoked Pagerduty API...' so it's less ambiguous?

Reply:
done